api/v1/db/oracledb/game-dao.js

```diff
@@ -168,11 +169,37 @@ const isMemberInGame = async (memberId, gameId) => {
   }
 };
 
+const cleanTableCardsByGameId = async (gameId, connection) => {
+  const sqlParams = [gameId];
+  const cleanCardSqlQuery = `
+  DELETE FROM TABLE_CARDS WHERE GAME_ID = :gameID
+  `;
+  await connection.execute(cleanCardSqlQuery, sqlParams);
+};
+
+
+const deleteGameByGameId = async (gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    await cleanTableCardsByGameId(gameId, connection);
+    await playerDao.deletePlayersByGameId(gameId, connection);
```

Comment:
Since the queries for deleting a game's table cards and deleting the players for a game shouldn't affect each other, could you perform these operations in parallel instead of in series?